dist/jsoneditor.js

```diff
@@ -1715,7 +1715,15 @@ JSONEditor.AbstractEditor = Class.extend({
     return null;
   },
   getTitle: function() {
-    return this.schema.title || this.key;
+      //Title collapsable form sections
+      if (this.hasOwnProperty("hide_add_button")){
+        if (this.parent !== undefined && !this.schema.title){
+          return this.parent.schema.definitions[this.key].plural_title;
```

Comment:
Should fall back on `title` or `key` here if the parent has no `plural_title` defined